iconservice/deploy/icon_builtin_score_loader.py

```diff
@@ -97,7 +97,7 @@ def _deploy_score(context: 'IconScoreContext',
             remove_path(score_deploy_path)
             # Copy builtin score source files from iconservice package to score_deploy_path.
             copytree(score_source_path_in_iconservice, score_deploy_path)
-        except FileExistsError:
+        except FileExistsError as e:
```

Comment:
The content seems to have intended to handle some errors.
However, it is not implemented.
Or you'd better get rid of it.